apps/docs/src/guide/wallets/setting-up-test-wallets.md

```diff
@@ -0,0 +1,13 @@
+# Setting up test wallets
+
+You'll often want to create one or more test wallets when testing your contracts. Here's how to do it.
+
+## Create a single wallet
+
+<<< @/../../docs-snippets/src/guide/wallets/access.test.ts#wallets{ts:line-numbers}
+
+## Setting up multiple test wallets
+
+If you need multiple test wallets, they can be set up using the `launchTestNode` utility. The wallets can be configured using the `walletsConfig` option. To understand the different configurations, check out the [walletsConfig](../testing//test-node-options.md#walletsconfig) in the test node options guide.
```

Comment:
Maybe this could be a bit more direct.

```suggestion
You can set up multiple test wallets using the `launchTestNode` utility via the `walletsConfigs` option.

To understand the different configurations, check out the [walletsConfig](../testing//test-node-options.md#walletsconfig) in the test node options guide.
```